akka-persistence-typed/src/test/scala/docs/akka/persistence/typed/InDepthPersistentBehaviorSpec.scala

```diff
@@ -75,6 +75,7 @@ object InDepthPersistentBehaviorSpec {
         case ChangeBody(newBody, replyTo) ⇒
           val evt = BodyChanged(state.postId, newBody)
           Effect.persist(evt).andThen { _ ⇒
+            println(s"Last sequence nr: ${ctx.lastSequenceNr}")
```

Comment:
remove println? Maybe as a log.debug instead?

Reply:
There is another println in the same method. Currently this code is used only from ShardingCompileOnlySpec (if I'm not wrong).

Reply:
I see, didn't see that on github at first; okey keeper then :)